build.proj

```diff
@@ -71,6 +71,18 @@
   <Import Project="dir.targets" />
   <Import Project="dir.traversal.targets" />
   <Import Project="$(ToolsDir)clean.targets" />
+  
+  <Target Name="CollectTemplateFiles" BeforeTargets="UpdateDependencies" >
+    <ItemGroup>
+      <ProjectJsonTemplateFiles Include="$(MSBuildThisFileDirectory)/tools-local/**/project.json.template" />
+      <ProjectJsonTemplateFiles Include="$(MSBuildThisFileDirectory)/src/**/project.json.template" />
+      <ProjectJsonFiles Remove="*.json" />
```

Comment:
Why do you need to remove "*.json" files? Are we all json.template files currently? Also where would the *.json files be getting added?